Settings.c

```diff
@@ -433,3 +441,7 @@ void Settings_invertSortOrder(Settings* this) {
       this->direction = 1;
    }
 }
+
+const Settings* Settings_getSettings(void) {
+   return settings;
+}
```

Comment:
No need for this new API, as the value is already handed everywhere where it's possibly needed.